<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #222831;
    }

    QLineEdit {
        background-color: #e0ffe0;
        color: #000000;
        font-size: 22px;
        border: 1px inset #888;
        padding: 8px;
        qproperty-alignment: AlignLeft;
    }

    QLineEdit#line_top {
        font-size: 16px;
        color: #444444;
    }

    QToolButton {
        background-color: #2d2d2d;
        color: #ffffff;
        border: 1px solid #555;
        border-radius: 6px;
        font-size: 18px;
        padding: 10px;
    }

    QToolButton[text="+"] ,
    QToolButton[text="-"] ,
    QToolButton[text="*"] ,
    QToolButton[text="/"] ,
    QToolButton[text="="] {
        color: #ffcc00;
        font-weight: bold;
        font-size: 24px;
    }

    QToolButton[text="AC"],
    QToolButton[text="←"],
    QToolButton[text="+/-"] {
        background-color: #bbbbbb;
        color: #000000;
        font-weight: bold;
    }

    QToolButton:hover {
        background-color: #3a3a3a;
    }

    QToolButton:pressed {
        background-color: #4a4a4a;
    }
   </string>
  </property>
  <widget class="QLineEdit" name="label_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_div">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_mul">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_dot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_eq">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_ac">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_ce">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_tog">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_top">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
